utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -382,6 +382,51 @@ Status PessimisticTransactionDB::CreateColumnFamily(
   return s;
 }
 
+Status PessimisticTransactionDB::CreateColumnFamilies(
+    const ColumnFamilyOptions& options,
+    const std::vector<std::string>& column_family_names,
+    std::vector<ColumnFamilyHandle*>* handles) {
+  InstrumentedMutexLock l(&column_family_mutex_);
```

Comment:
I wonder if lock acquisition can be postponed after line 394, but we don't have to address this in this PR, given that `CreateColumnFamily`'s behavior.